gramps/plugins/lib/libgedcom.py

```diff
@@ -1129,14 +1129,12 @@ def __init__(self):
 
         try:
             filepath = os.path.join(DATA_DIR, "gedcom.xml")
-            ged_file = open(filepath, "rb")
+            with open(filepath, "rb") as ged_file:
+                parser = GedInfoParser(self)
```

Comment:
I'm not sure about having all of this in the try block. And not giving any indication of error if there is a problem. Wonder what that try block is supposed to be doing?